src/Form/Type/PriceFilterType.php

```diff
@@ -57,7 +57,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                         'message' => 'bitbag_sylius_elasticsearch_plugin.max_price_numeric',
                     ]),
                     new PositiveOrZero([
-                        'message' => 'bitbag_sylius_elasticsearch_plugin.min_price_positive_or_zero',
+                        'message' => 'bitbag_sylius_elasticsearch_plugin.max_price_positive_or_zero',
```

Comment:
Is this key present in translations? I don't see such entry: https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/blob/master/src/Resources/translations/messages.en.yml

Reply:
Yes, of course, it's validator message: https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/blob/623ff1f4bd8ac469bfed46534b53beefd916e0d6/src/Resources/translations/validators.en.yml#L5